### ISSUE TITLE
Map options - add WMS layer not working for "GA"

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/spatial-hub#249]
Using the "server examples" option, selecting "GA" option does not work (no layers are shown below), other other options do work as expected.

JS error appears in the console:

```
application.unminified.js:2275 TypeError: Cannot read property 'WMS_Capabilities' of null
    at application.unminified.js:12756
    at application.unminified.js:2253
    at application.unminified.js:2288
    at t.$eval (application.unminified.js:2302)
    at t.$digest (application.unminified.js:2299)
    at t.$apply (application.unminified.js:2303)
    at g (application.unminified.js:2254)
    at v (application.unminified.js:2259)
    at XMLHttpRequest.B.onload (application.unminified.js:2260)
```
I also tried pasting a WMS `GetCapabilities` URL (that works in a browser) but got not output (no JS error this time):

https://data.gov.au/geoserver/other-state-boundary-psma-administrative-boundaries/wms?request=GetCapabilities

**Comment by adam-collins** [comment]
Released in 0.3.4

**Comment by Tasilee** [comment]
url not permitted[http://services.ga.gov.au/gis/services/GA_Surface_Geology/MapServer/WMSServer?request=GetCapabilities&service=WMS may not in proxy list!]
